analyzers/tests/SonarAnalyzer.UnitTest/TestCases/DisposableNotDisposed.CSharp9.cs

```diff
@@ -50,13 +58,25 @@ record MyRecord
     private FileStream field_fs3 = new FileStream(@"c:\foo.txt", FileMode.Open); // Noncompliant {{Dispose 'field_fs3' when it is no longer needed.}}
             // Noncompliant@-1 - duplicate
     private FileStream field_fs4 = new FileStream(@"c:\foo.txt", FileMode.Open); // Compliant - disposed
+    private FileStream field_fs5 = new(@"c:\foo.txt", FileMode.Open); // Noncompliant {{Dispose 'field_fs5' when it is no longer needed.}}
+    // Noncompliant@-1 - duplicate
+
+    private FileStream backing_field1;
+    public FileStream Prop1
+    {
+        init
+        {
+            backing_field1 = new FileStream("", FileMode.Open); // Noncompliant
+            // Noncompliant@-1 - duplicate
```

Comment:
Let's align this one with the one above, this was hard to read (same below)

Reply:
There's quite a lot of misalignments in the file, can you fix them too? 
The `(same below)` meant below, in the same file :)

Reply:
Saw this comment just a minute ago. Having strange github sync issues, reloading the page doesn't help to get new comments.